server/util-server.js

```diff
@@ -89,6 +90,52 @@ exports.pingAsync = function (hostname, ipv6 = false) {
     });
 };
 
+exports.mqttAsync = function (hostname, topic, okMessage, options = {}) {
```

Comment:
Just come back to test this pr, I discovered that mqttAsync could be running into a dead loop state, especially if the server is down.

Reply:
I'm thinking of adding a reject when timeout is triggered and then using this "timeout" keyword to bypass the "DOWN" label.